Annotato/Annotato/Persistence/PersistenceService/OfflinePersistenceService/OfflinePersistenceService+AnnotationsPersistence.swift

```diff
@@ -17,7 +17,23 @@ extension OfflinePersistenceService: AnnotationsPersistence {
         return await localPersistence.annotations.deleteAnnotation(annotation: annotation)
     }
 
-    func createOrUpdateAnnotations(annotations: [Annotation]) -> [Annotation]? {
-        localPersistence.annotations.createOrUpdateAnnotations(annotations: annotations)
+    func createOrUpdateAnnotation(annotation: Annotation) async -> Annotation? {
+        if LocalAnnotationEntityDataAccess.read(annotationId: annotation.id,
```

Comment:
I think the `OfflinePersistenceService` should not be made aware of the DataAccess layer. Instead, it should just go through `localPersistence.annotations.getAnnotation` or something. What do you think?